texthero/representation.py

```diff
@@ -320,25 +358,23 @@ def tfidf(
         If float, the parameter represents a proportion of documents, integer
         absolute counts.
 
-    return_features_names : Boolean, False by Default
-        If True, return a tuple (*count_series*, *features_names*)
-
-
     Examples
     --------
     >>> import texthero as hero
     >>> import pandas as pd
     >>> s = pd.Series(["Hi Bye", "Test Bye Bye"]).pipe(hero.tokenize)
-    >>> hero.tfidf(s, return_feature_names=True)
-    (document
-    0    [1.0, 1.4054651081081644, 0.0]
-    1    [2.0, 0.0, 1.4054651081081644]
-    dtype: object, ['Bye', 'Hi', 'Test'])
+    >>> hero.tfidf(s) # doctest: +SKIP
```

Comment:
Why are we skipping the doctest? What's the output of `hero.tfidf(s)` there?

Reply:
No idea, removed it